documentation/frontend/0000-use-markdown-architectural-decision-records.md

```diff
@@ -0,0 +1,26 @@
+# Use Markdown Architectural Decision Records
```

Comment:
Should all these ADRs be moved into our ADR directory and merged with our ADRs in terms of numbering, etc?

Reply:
Sure, I'll move those over.